.env.docker

```
@@ -0,0 +1,30 @@
+# Docker Environment Variables
+# Copy this to .env.local and update with your actual values
+
+# Database
+DATABASE_URL="postgresql://outleads_user:outleads_secure_password@postgres:5432/outleads?schema=public"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)